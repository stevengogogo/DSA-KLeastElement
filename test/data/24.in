10 50000
-7537 -4730 -1649 1113 482 -8704 2781 1330 -4089 -5512
Reverse 1 6
Insert 3 3722
Query 8 10 3
Insert 12 6308
Insert 4 -944
Insert 11 -7115
Reverse 4 8
Reverse 2 7
Insert 12 -5381
Reverse 1 7
Reverse 14 15
Query 6 13 1
Reverse 14 14
Insert 13 1437
Query 4 5 1
Reverse 13 15
Insert 16 6978
Query 12 12 1
Delete 10
Delete 11
Delete 10
Query 12 13 1
Query 9 11 1
Insert 9 -3872
Insert 1 -6839
Query 11 15 4
Insert 10 5029
Delete 16
Insert 16 8746
Insert 13 -9766
Delete 14
Delete 9
Query 4 8 5
Insert 10 2068
Delete 11
Query 4 8 3
Delete 12
Delete 2
Delete 4
Delete 13
Delete 5
Query 5 5 1
Reverse 10 10
Delete 2
Reverse 5 5
Delete 3
Query 1 6 1
Query 9 9 1
Query 1 7 2
Delete 7
Query 8 8 1
Insert 2 4756
Insert 9 1171
Delete 8
Reverse 4 9
Reverse 3 8
Query 7 9 1
Delete 5
Insert 8 4408
Query 8 8 1
Query 1 6 2
Delete 3
Reverse 3 3
Query 4 7 4
Delete 2
Query 3 5 3
Delete 5
Query 5 5 1
Reverse 1 5
Insert 5 -8739
Insert 4 -1732
Insert 8 -7734
Query 9 9 1
Delete 2
Query 6 6 1
Insert 6 -6872
Delete 3
Query 3 4 1
Reverse 8 8
Query 7 8 2
Query 5 6 2
Insert 4 -8042
Delete 7
Delete 4
Reverse 4 4
Delete 5
Reverse 5 5
Query 1 1 1
Delete 2
Query 1 3 1
Query 4 5 2
Delete 2
Insert 2 -8550
Reverse 1 1
Delete 1
Delete 4
Insert 1 -9667
Reverse 4 4
Reverse 3 4
Query 4 4 1
Delete 2
Insert 2 -2144
Reverse 1 4
Insert 1 -9648
Query 2 4 3
Delete 3
Query 3 3 1
Query 1 4 2
Query 2 2 1
Query 3 4 1
Insert 2 3866
Query 4 4 1
Query 4 4 1
Delete 3
Reverse 1 4
Delete 2
Insert 4 5435
Query 2 4 3
Insert 4 -9378
Insert 3 -2260
Reverse 3 3
Delete 5
Reverse 4 5
Query 2 5 4
Delete 5
Insert 5 3560
Reverse 1 2
Insert 5 -983
Delete 2
Reverse 1 3
Delete 3
Reverse 4 4
Reverse 3 4
Delete 1
Insert 3 -5893
Delete 1
Reverse 2 2
Reverse 1 1
Reverse 2 2
Reverse 1 2
Query 3 3 1
Reverse 3 3
Delete 3
Delete 2
Insert 2 -290
Reverse 1 2
Delete 2
Query 1 1 1
Reverse 1 1
Delete 1
