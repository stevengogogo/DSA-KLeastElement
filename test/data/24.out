2781
-8704
-4730
-5381
1437
-4089
6308
1113
-4730
-8704
-8704
8746
-7537
8746
-8704
4408
1171
8746
8746
4408
-8704
-6839
-8739
-7734
-6839
4408
-8739
-7734
-8739
-2144
-2144
-9648
-8739
-9667
-2144
-2144
5435
5435
-983
-290
